list/LIST_CONSULTATION_README.md

```diff
@@ -0,0 +1,209 @@
+# List (Consultation) Mapping
+
+## XML HL7 > JSON FHIR
+
+A List (Consultation) is created from mapped [Encounters](../encounters/README.md). Where the EHR extract is used it is the root `EhrExtract` of the XML.
+This is a top-level profile and represents the whole structured consultation
+
+| Mapped to (JSON FHIR List field) | Mapped from (XML HL7 / other source)                                                  |
+|----------------------------------|---------------------------------------------------------------------------------------|
+| id                               | `Encounter / id [@root]`                                                              |
+| meta.profile\[0]                 | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-List-1"` |
+| status                           | fixed value = `current`                                                               |
+| mode                             | fixed value = `snapshot`                                                              |
+| title                            | `Encounter / type`                                                                    |
+| code.coding\[0].system           | fixed value = `http://snomed.info/sct`                                                |
+| code.coding\[0].code             | fixed value = `325851000000107`                                                       |
+| code.coding\[0].display          | fixed value = `Consultation`                                                          |
+| subject                          | this a reference to mapped [Patient](../patient/README.md) from `Encounter /          |
```

Comment:
Missing closing backtick?  Also feels a bit janky.  Maybe "this is a reference to the encounters mapped patient"?

Reply:
updated this accordingly

Reply:
corrected